docs/source/llms/chat-model-guide/index.rst

```diff
@@ -0,0 +1,802 @@
+        - **Logged within the MLmodel file** and will be visible within the artifact viewer in the MLflow UI
+
+        The ``model_config`` values that are submitted for our ``BasicAgent`` example within this tutorial can be seen within the logged model's
+        ``MLmodel`` file in the UI:
+
+        .. figure:: ../../_static/images/llms/chat-model-guide/model-config-in-ui.png
+            :alt: Model configuration in the MLflow UI
+            :width: 50%
+            :align: center
+
+
+    .. tab:: Standardization
+
+        .. raw:: html
+
+            <h3>Standardization for GenAI Models</h3>
+        
+        |
+
+        One of the more complex tasks associated with deploying a GenAI application with MLflow arises when attempting to build a custom implementation
+        that is based on subclassing the :py:class:`mlflow.pyfunc.PythonModel` abstraction. 
+
+        While ``PythonModel`` is recommended for custom Deep Learning and traditional Maching Learning models (such as ``sklearn`` or ``torch`` models that require
+        additional processing logic apart from that of a base model), there are internal manipulations of the input data that occur
+        when serving these models that introduce unneccessary complications with GenAI applications. 
+
+        Due to the fact that DL and traditional ML models largely rely on structured data, when input data is passed via a REST interface for model serving,
+        the ``PythonModel`` implementation will convert JSON data into ``pandas.DataFrame`` or ``numpy`` objects. This conversion creates a confusing and difficult to
+        debug scenario when using GenAI models. GenAI implementations generally deal exclusively with JSON-conformant data structures and have no tabular 
+        representation that makes intuitive sense, thereby creating a frustrating and complex conversion interface needed to make application deployment function
+        correctly. 
+
+        To simplify this problem, the :py:class:`mlflow.pyfunc.ChatModel` class was created to provide a simpler interface for handling of the data
+        passed into and returned from a call to the ``predict()`` method on custom Python models serving GenAI use cases. 
+
+        In the example tutorial code below, we subclass ``ChatModel`` in order to utilize this simplified interface with its immutable input and output 
+        formats. Because of this immutability, we don't have to reason about model signatures, and can instead directly use API standards that have 
+        been broadly accepted throughout the GenAI industry. 
+
+        To illustrate why it is preferred to use ``ChatModel`` as a super class to custom GenAI implementations in MLflow, here is the signature that
+        would otherwise need to be defined and supplied during model logging to conform to the ``OpenAI`` API spec as of September 2024:
+
+        **Input Schema** as a ``dict``:
+
+        .. code-block:: python
```

Comment:
it's actually even worse than this if you include tool calling, but probably the user understands the point 😄

Reply:
great point - added a note about this to reinforce the message